src/Compilers/CSharp/Test/Symbol/Symbols/AssemblyAndNamespaceTests.cs

```diff
@@ -112,11 +112,11 @@ struct S {}
     }
 }
 ";
-            var comp1 = CSharpCompilation.Create(assemblyName: "Test", options: TestOptions.ReleaseExe,
+            var comp1 = CSharpCompilation.Create(assemblyName: "Test", options: TestOptions.DebugExe,
```

Comment:
I don't know why these changed from Release to Debug.

Reply:
@RikkiGibson, the original code from master was using the default, which is debug. In my first commit I used Release by mistake, then I fixed that. Probably to better to view the diff from all commits.